appengine-java11/oauth2/pom.xml

```diff
@@ -28,108 +28,109 @@
       Dependent-jars are created and added to
       ${project.build.directory}/appengine-staging -->
     <dependencies>
-        <!-- Dependency from folder `../appengine-simple-jetty-main` -->
-        <!-- See README for instructions for more information -->
-        <dependency>
-            <groupId>com.example.appengine.demo</groupId>
-            <artifactId>simple-jetty-main</artifactId>
-            <version>1</version>
-            <scope>provided</scope>
-        </dependency>
-
-        <!-- OAuth 2.0 packages -->
-        <!-- [START gae_java11_oauth2_dependencies] -->
-        <dependency>
-            <groupId>com.google.oauth-client</groupId>
-            <artifactId>google-oauth-client</artifactId>
-            <version>1.28.0</version>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>com.google.oauth-client</groupId>
-            <artifactId>google-oauth-client-servlet</artifactId>
-            <version>1.28.0</version>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>com.google.apis</groupId>
-            <artifactId>google-api-services-oauth2</artifactId>
-            <version>v2-rev131-1.23.0</version>
-            <scope>provided</scope>
-        </dependency>
-        <!-- [END gae_java11_oauth2_dependencies] -->
-
-        <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <version>4.0.1</version>
-            <type>jar</type>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>jstl</groupId>
-            <artifactId>jstl</artifactId>
-            <version>1.2</version>
-            <scope>provided</scope>
-        </dependency>
+      <!-- Dependency needs to be locally install from directory -->
+      <!-- `java-docs-samples/appengine-java11/appengine-simple-jetty-main' -->
+      <!-- See the README for more information -->
+      <dependency>
+          <groupId>com.example.appengine.demo</groupId>
+          <artifactId>simple-jetty-main</artifactId>
+          <version>1</version>
+          <scope>provided</scope>
+      </dependency>
+
+      <!-- OAuth 2.0 packages -->
+      <!-- [START gae_java11_oauth2_dependencies] -->
+      <dependency>
+        <groupId>com.google.oauth-client</groupId>
+        <artifactId>google-oauth-client</artifactId>
+        <version>1.28.0</version>
+        <scope>provided</scope>
+      </dependency>
+
+      <dependency>
+        <groupId>com.google.oauth-client</groupId>
+        <artifactId>google-oauth-client-servlet</artifactId>
+        <version>1.28.0</version>
+        <scope>provided</scope>
+      </dependency>
+
+      <dependency>
+        <groupId>com.google.apis</groupId>
+        <artifactId>google-api-services-oauth2</artifactId>
+        <version>v2-rev131-1.23.0</version>
```

Comment:
com.google.apis:google-api-services-oauth2:v2-rev20190313-1.28.0 . ??

Reply:
done. Next time I will remember to check the dependencies. Thanks!